pythonforandroid/python.py

```diff
@@ -404,22 +438,25 @@ def build_arch(self, arch):
         build_dir = join(recipe_build_dir, self.build_subdir)
         ensure_dir(build_dir)
 
-        if not exists(join(build_dir, 'python')):
-            with current_directory(recipe_build_dir):
-                # Configure the build
-                with current_directory(build_dir):
-                    if not exists('config.status'):
-                        shprint(
-                            sh.Command(join(recipe_build_dir, 'configure')))
+        with current_directory(recipe_build_dir):
+            # Configure the build
+            with current_directory(build_dir):
+                if not exists('config.status'):
+                    shprint(
+                        sh.Command(join(recipe_build_dir, 'configure')))
 
-                # Create the Setup file. This copying from Setup.dist
-                # seems to be the normal and expected procedure.
-                shprint(sh.cp, join('Modules', 'Setup.dist'),
-                        join(build_dir, 'Modules', 'Setup'))
+            # Create the Setup file. This copying from Setup.dist
+            # seems to be the normal and expected procedure.
+            shprint(sh.cp, join('Modules', 'Setup.dist'),
+                    join(build_dir, 'Modules', 'Setup'))
 
-                shprint(sh.make, '-j', str(cpu_count()), '-C', build_dir)
-        else:
-            info('Skipping {name} ({version}) build, as it has already '
-                 'been completed'.format(name=self.name, version=self.version))
+            shprint(sh.make, '-j', str(cpu_count()), '-C', build_dir)
+
+        # Since we got different python's executable names depending on the
+        # build platform, we add a a debug message which prints the generated
+        # files/folders, in case that we need that...
+        python_dir = join(self.get_build_dir(arch.arch), self.build_subdir)
+        info("Files/folders in python's build directory are:")
+        shprint(sh.ls, python_dir)
```

Comment:
@inclement, [about the generated python.exe for Mac OSX](https://github.com/kivy/python-for-android/pull/1951#pullrequestreview-270513660)

I read it in the [build docs for python](https://github.com/python/cpython/blob/3.7/README.rst#build-instructions). Now I also added an `ls command` to the hostpython build so we see the generated files/folders. I checked it by enabling a debug build in ci for OSX in another branch and `python.exe` is there...:thinking:...should we remove those debug lines?